docs/tutorials/01_introduction_cloud_runtime.ipynb

```diff
@@ -0,0 +1,734 @@
+{
```

Comment:
<p>Do we want to include <strong>specific</strong> execution time limits here? Don't they differ per user, device and program type?</p><p> </p><p>Might be hard to keep this tutorial up-to-date when the defaults change. So I suggest to just mention that there is a max. execution time after which the execution will be forcibly terminated.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-ibm-runtime/pull/97/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>It's meant to point out the "maximum execution time" field printed out in the previous cell. But yeah it could get out of sync. Removed in <a href="https://github.com/Qiskit/qiskit-ibm-runtime/pull/97/commits/b7787c703d410dbb9135838d47530b4c69f7d15a" target="_blank">b7787c7</a>. </p>